pkg/route/api/allocator/allocator.go

```diff
@@ -0,0 +1,30 @@
+package allocator
```

Comment:
@smarterclayton is this something you think warrants an interface so it can have different implementations in the future (perhaps for different allocation strategies)?